### DISCUSSION TITLE
WHY MY ACCOUNT FLAGGED AS A SPAM AFTER I USED MY SCHOOL MAIL FOR GITHUB STUDENT ??

### DISCUSSION THREAD

**Discussion by Faadddi123** [community/community#116781] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Recently, I added my school email to github so I could get a github student. I followed instructions, and after some time, I got approved, but a week or so later, I got my account flagged as spam.
I use my github daily, so it really stops my work now.
I need this to be solved as soon as possible, but I don't know how.

the Github affected username is [this](https://github.com/WailBoukhari) .

**Comment by giovannivicentin** [comment]
Hi faaaddd,

I'm sorry to hear about the trouble you're experiencing with your GitHub account being flagged. This situation can indeed be frustrating, especially when it impacts your daily work and studies. Here's a step-by-step guide on how you might address this issue:

1. **Review GitHub's Policies**: Start by reviewing GitHub's community guidelines and terms of service to ensure that your usage aligns with their policies. Sometimes accounts are flagged due to misunderstandings or unintended violations.

2. **Contact GitHub Support**: The most direct way to resolve this is to contact GitHub Support. You can do this by visiting the GitHub Support page and submitting a request. Make sure to choose the "Account Questions" category or something similar that fits your issue.

   - In your communication with GitHub, be clear and concise. Explain that your account was flagged shortly after being approved for GitHub Student Developer Pack and that you rely on GitHub for your daily work and school projects. Mention any relevant details, such as the last activities you performed before the account was flagged.
   
   - Provide your GitHub username but avoid sharing sensitive personal information publicly or in forums.

3. **Be Patient and Responsive**: After you've submitted your request, it might take some time for GitHub support to review your case. Be patient, and keep an eye on your email (including spam/junk folders) for a response from GitHub. They may ask for additional information to verify your identity or clarify your account activities.

4. **Review Your Account Activities**: While waiting for a response, review your recent activities on GitHub. If there are any projects or actions that could potentially be misconstrued as spammy behavior (like automated or excessive repository creations, issues, comments, etc.), make a note of them. Being able to explain or clarify your actions can be helpful in your communications with GitHub.

5. **Prepare for Future Prevention**: Once your account is back in good standing, consider taking steps to prevent similar issues. This might include familiarizing yourself further with GitHub's best practices, managing your activities to avoid triggering spam filters, and regularly reviewing account settings and security.

If you need further clarification on any of these steps or additional assistance, please don't hesitate to ask. I'm here to help you navigate through this.

Best

**Comment by ghost** [comment]
Hey @Faadddi123, this is exactly what happened to me as well when I added my school email. Many people on community are suggesting to try github support, but it didn't help me:

https://github.com/orgs/community/discussions/121640

I have created similar ticket and described there what I have tried. Maybe it can help you, but I am still with this issue and as far as I can see you are too
